picard/ui/options/interface.py

```diff
@@ -76,6 +76,7 @@ class InterfaceOptionsPage(OptionsPage):
         BoolOption("setting", "show_menu_icons", True if not IS_MACOS else False),  # On macOS it is not common that the global menu shows icons
         BoolOption("setting", "builtin_search", True),
         BoolOption("setting", "use_adv_search_syntax", False),
+        BoolOption("setting", "show_new_user", True),
```

Comment:
I suggest we use a more explicit name for the option: `show_new_user_dialog`

Patch -> https://github.com/rdswift/picard/pull/16